homeassistant/components/generic/strings.json

```diff
@@ -0,0 +1,77 @@
+    "error": {
+      "unknown": "[%key:common::config_flow::error::unknown%]",
+      "already_exists": "A camera with these URL settings already exists.",
+      "unable_still_load": "Unable to load valid image from still image URL (e.g. invalid host, URL or authentication failure).  Review log for more info.",
+      "no_still_image_or_stream_url": "You must specify at least a still image or stream URL",
+      "invalid_still_image": "URL did not return a valid still image",
+      "stream_file_not_found": "File not found while trying to connect to stream (is ffmpeg installed?)",
+      "stream_http_not_found": "HTTP 404 Not found while trying to connect to stream",
+      "timeout": "Timeout while loading URL",
+      "stream_no_route_to_host": "Could not find host while trying to connect to stream",
+      "stream_io_error": "Input/Output error while trying to connect to stream. Wrong RTSP transport protocol?",
+      "stream_unauthorised": "Authorisation failed while trying to connect to stream",
+      "stream_not_permitted": "Operation not permitted while trying to connect to stream. Wrong RTSP transport protocol?",
+      "stream_novideo": "Stream has no video"
+    },
+    "abort": {
+      "single_instance_allowed": "[%key:common::config_flow::abort::single_instance_allowed%]",
+      "no_devices_found": "[%key:common::config_flow::abort::no_devices_found%]"
+    },
+    "step": {
+      "user": {
+        "description": "Enter the settings to connect to the camera.",
+        "data": {
+          "still_image_url": "Still Image URL (e.g. http://...)",
+          "stream_source": "Stream Source URL (e.g. rtsp://...)",
+          "rtsp_transport": "RTSP transport protocol",
+          "authentication": "Authentication",
+          "limit_refetch_to_url_change": "Limit refetch to url change",
+          "password": "[%key:common::config_flow::data::password%]",
+          "username": "[%key:common::config_flow::data::username%]",
+          "content_type": "Content Type",
+          "framerate": "Frame Rate (Hz)",
+          "verify_ssl": "[%key:common::config_flow::data::verify_ssl%]"
+        }
+      },
+      "confirm": {
+        "description": "[%key:common::config_flow::description::confirm_setup%]"
+      }
+    }
+  },
+  "options": {
+    "step": {
+      "init": {
+        "data": {
+          "name": "Device Name",
+          "still_image_url": "Still Image URL (e.g. http://...)",
+          "stream_source": "Stream Source URL (e.g. rtsp://...)",
+          "rtsp_transport": "RTSP transport protocol",
+          "authentication": "Authentication",
+          "limit_refetch_to_url_change": "Limit refetch to url change",
+          "password": "[%key:common::config_flow::data::password%]",
+          "username": "[%key:common::config_flow::data::username%]",
+          "content_type": "Content Type",
+          "framerate": "Frame Rate (Hz)",
+          "verify_ssl": "[%key:common::config_flow::data::verify_ssl%]"
```

Comment:
Please make the non-common ones references to the other strings so we do not have to translate them twice
example
https://github.com/home-assistant/core/blob/dev/homeassistant/components/homekit/strings.json#L12

Reply:
Done 176cc88c.